variables.tf

```
@@ -4,30 +4,43 @@
 # You must provide a value for each of these parameters.
 # ------------------------------------------------------------------------------
 
-variable "subnet_id" {
-  type        = string
-  description = "The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0)."
+variable "auditors" {
+  type        = map(map(list(string)))
```

Just as a thought but would this be better defined as:

```suggestion
  type        = map(object({ roles = list(string) }))
```

---

You are correct!  Please see https://github.com/cisagov/cyhy-auditor-iam/pull/3/commits/0be4c7ac003bd92310b7e2b612b4e9f8747d7f13.